drivers/timers/Kconfig

```diff
@@ -366,13 +366,21 @@ if WATCHDOG_AUTOMONITOR
 
 choice
 	prompt "Auto-monitor keepalive by"
-	default WATCHDOG_AUTOMONITOR_BY_TIMER
+	default WATCHDOG_AUTOMONITOR_BY_WDOG
 
 config WATCHDOG_AUTOMONITOR_BY_CAPTURE
 	bool "Capture callback"
 
+config WATCHDOG_AUTOMONITOR_BY_ONESHOT
+	bool "Oneshot callback"
+	depends on ONESHOT
+
 config WATCHDOG_AUTOMONITOR_BY_TIMER
 	bool "Timer callback"
+	depends on TIMER
```

Comment:
Ditto, also it could be useful to give some hints what/how he will configue the timer later

Reply:
Done.